guides/doc-Administering_Red_Hat_Satellite/topics/Renaming_a_Capsule_Server.adoc

```diff
@@ -6,6 +6,18 @@ This procedure ensures that you update all references to the new host name.
 
 .Prerequisites
 
+[WARNING]
+====
+Until BZ1829115 is resolved, you must edit the `usr/share/katello/hostname-change.rb` file on {SmartProxyServer} and comment out the following lines before attempting to rename {SmartProxyServer}:
```

Comment:
Sorry, Ian I should have mentioned this, we always provide a clickable link to the bug also.